nixos/modules/services/networking/wpa_supplicant.nix

```diff
@@ -47,6 +66,16 @@ in {
         description = "Force a specific wpa_supplicant driver.";
       };
 
+      scanOnLowSignal = mkOption {
+        type = types.bool;
+        default = true;
```

Comment:
I think this should be disabled by default.

Reply:
I'm enabling this by default because it makes roaming "just work". Without this users would probably just assume (as in https://github.com/NixOS/nixpkgs/issues/105560) roaming in wpa_supplicant doesn't work
It can be debated, but I think the benefit of working roaming outweights saving battery percents, though I haven't tried to quantify the impact.

Reply:
I don't think it impacts battery at all. It will be very confusing for someone who has to move their laptop and `wpa_supplicant` magically changes networks etc.

Reply:
wpa_supplicant (and every other 802.11 client probably) is already doing this: it rescans and connects to the best network available. It's just that, to save battery, it's doing that only if the signal drops to zero.

Reply:
What about enabling it by default only for minimal ISO? That would solve #105560 while keeping the existing behavior for normal installation.

Reply:
The issue is about using iwd to save space in the minimal install, I don't think enabling roaming only here would help much: when you installing NixOS you're likely not walking around. Proper roaming is useful in you main machine, not the installer.